src/salmon/quant/config.vsh.yaml

```diff
@@ -0,0 +1,582 @@
+name: salmon_quant
+namespace: salmon
+description: | 
+  Salmon is a tool for wicked-fast transcript quantification from RNA-seq data. It can either make use of pre-computed alignments (in the form of a SAM/BAM file) to the transcripts rather than the raw reads, or can be run in the mapping-based mode. 
+keywords: ["Transcriptome", "Quantification"]
+links:
+  homepage: https://salmon.readthedocs.io/en/latest/salmon.html
+  documentation: https://salmon.readthedocs.io/en/latest/salmon.html
+  repository: https://github.com/COMBINE-lab/salmon
+references: 
+  doi: "doi:10.1038/nmeth.4197"
+license: GPL-3.0 
+requirements:
+  commands: [ salmon ]
+
+argument_groups:
+  - name: Common input options
+    arguments:
+      - name: --lib_type
+        alternatives: ["-l"]
+        type: string
+        description: |
+          Format string describing the library.
+        required: false
+        example: 'A'
+        choices: ['A', 'U,', 'SF', 'SR', 'IU', 'IS', 'ISF', 'ISR', 'OU', 'OS', 'OSF', 'OSR', 'MU', 'MS', 'MSF', 'MSR']
+  
+  - name: Mapping input options
+    arguments:
+      - name: --index
+        alternatives: ["-i"]
+        type: file
+        description: |
+          Salmon index.
+        required: true
+        example: transcriptome_index
+      - name: --unmated_reads
+        alternatives: ["-r"]
+        type: file
+        description: |
+          List of files containing unmated reads of (e.g. single-end reads).
+        required: false
+        multiple: true
+        multiple_sep: " "
```

Comment:
Could you change all multiple_sep's into `;`?

This is because other values for `multiple_sep` such as `:`, ` `, or `,` might not always work for what is otherwise a valid file path.

Reply:
Changing it to `;` doesn't work for some reason but I haven't been able to figure out why exactly.